certificate-print-service/build.gradle

```diff
@@ -0,0 +1,100 @@
+plugins {
+    id 'org.springframework.boot' version '3.4.0'
+    id 'io.spring.dependency-management' version '1.1.4'
+    id 'org.sonarqube' version '5.1.0.4882'
+    id "org.owasp.dependencycheck" version "10.0.4"
+    id 'test-report-aggregation'
+    id 'application'
+}
+
+ext {
+    ineraMavenRepository = 'https://nexus.drift.inera.se/repository/it-public/'
+    nvdMirror = 'https://vulnz-nvd-mirror-devtest.intyg.nordicmedtest.se'
+    springbootMavenBomVersion = '3.4.0'
+    logbackEcsEncoderVersion = '1.6.0'
```

Comment:
We don't need logbackEcsEncoder when we are using spring boots structured logging.